ass_man/views.py

```diff
@@ -224,9 +227,65 @@ def destroy(self, request, *args, **kwargs):
             return Response('Cannot delete this rack as it contains instances: ' +
                             ', '.join(offending_instances),
                             status=status.HTTP_400_BAD_REQUEST)
-        super().destroy(self, request, *args, **kwargs)
+        return super().destroy(self, request, *args, **kwargs)
 
     # New Actions
+    @action(detail=False, methods=['POST', 'DELETE'])
+    def many(self, request, *args, **kwargs):
+        try:
+            srn = request.data['start_rack_num']
+            ern = request.data['end_rack_num']
+        except KeyError:
+            return Response('Invalid rack range- must specify start and end rack number',
+                            status=status.HTTP_400_BAD_REQUEST)
+        try:
+            s_letter = srn[0].upper()
+            e_letter = ern[0].upper()
+            s_number = int(srn[1:])
```

Comment:
what if rack number is AA12? @rishi-tripathy

Reply:
won't work